lib/order_book.js

```diff
@@ -220,6 +220,10 @@ class OrderBook extends EventEmitter {
       ? entry.length === 4 ? 2 : 1
       : 0
     const numEntry = entry.map((x) => Number(x))
+    const isValid = numEntry.every(value => Number.isFinite(value))
+
+    if (!isValid) throw new Error('the entry is not valid (should contain only numbers)')
```

Comment:
we should return instead of throwing error here especially since this class is based on event emitter

Reply:
@vigan-abd what should be returned if the input is invalid?

Reply:
based on one below https://github.com/bitfinexcom/bfx-api-node-models/pull/68/files/9619439809925c1bc715d5ace30923d6f1a01be9#diff-a89d352f77a35238429009e95175b27bae98d43f1face3eddcbbf289b592ad32R262-R265 I think we should simply